api/server.ts

```diff
@@ -3,34 +3,33 @@ import dotenv from 'dotenv';
 import express from 'express';
 import { Server } from 'typescript-rest';
 import createLogger, { initLogger } from './utils/logger';
+import { ENV, PORT, LOG_LEVEL } from './utils/constants';
```

Comment:
if this is imported before calling `dotenv.config(...)`, then the values won't be updated with what's in the `.env` file. (That's why I had a set of imports _after_ calling that function)

Reply:
Ok I can update the npm scripts then to do the dot env in the console before compilation. That way the ENV variable will be populated anywhere within the app.

Reply:
ooh didn't know you could do that! Good call

Reply:
Yea the `dot-env` package has a cli so you can just do something like `dotenv --config=config/path.json && jest` or something similar